dist/metricsgraphics.css

```diff
@@ -226,6 +246,50 @@ tspan.hist-symbol {
     fill: #5c5c5c;
 }
 
```

Comment:
I think we should define a color called `.mg-line-color` here (repeating pattern for the other types) with a stroke color of black. We could then give the line (or whatever other object you're manipulating) that class and it should appear in that color as long as there isn't class overriding it (which it would be default for up to 10 lines, and possibly further on that if the user creates a user-defined style).

Reply:
ok, I'll try and implement the same :+1: